packages/flutter/lib/src/material/tabs.dart

```diff
@@ -832,6 +833,12 @@ class TabBar extends StatefulWidget implements PreferredSizeWidget {
   /// Defaults to matching platform conventions.
   final ScrollPhysics? physics;
 
+  /// Defines the [InkWell]'s `borderRadius` property.
```

Comment:
The doc should not reference something the developer may not know about, like `TabBar` using `InkWell`. Try documenting it like below.

```dart
/// The clipping radius of splashes that extend outside the bounds of the tab.
```

Try reading docs of similar API from other parts of the framework to get a better idea.